js-app.js

```diff
@@ -0,0 +1,167 @@
+const menuBar = document.querySelector('#menu-bar');
+const xIcon = document.querySelector('#x-icon');
+const popupList = document.querySelector('.popup-list');
+
+xIcon.style.display = 'none';
+popupList.style.display = 'none';
+
+menuBar.addEventListener('click', () => {
+  xIcon.style.display = 'block';
+  popupList.style.display = 'block';
+  menuBar.style.display = 'none';
+});
+
+xIcon.addEventListener('click', () => {
+  menuBar.style.display = 'block';
+  popupList.style.display = 'none';
+  xIcon.style.display = 'none';
+});
```

Comment:
[OPTIONAL] You can merge your app.js and js-app.js file since they have repeating code.
<img width="477" alt="Screenshot 2023-04-22 at 6 35 04 PM" src="https://user-images.githubusercontent.com/25887253/233793614-fb3f7871-b0f3-483c-98e9-02034b024afa.png">
<img width="477" alt="Screenshot 2023-04-22 at 6 35 10 PM" src="https://user-images.githubusercontent.com/25887253/233793616-6556393a-9bcc-40ed-af7d-d305631f292f.png">